### PR TITLE
[FLINK-13343][docs-zh] Translate "Contribute Code" page into Chinese

### PR DESCRIPTION
**Pull request by highfei2011**
Translate "Contribute Code" page into Chinese.
I have checked and passed.Please take a look . Thanks @wuchong  @xccui  @klion26  .

### CODE REVIEW

**Comment by klion26** [comment]
thanks for your contribution, you don't need to update the html file.

**Comment by highfei2011** [comment]
Thanks @klion26 , I will recover this change.

File: content/zh/contributing/contribute-code.html [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-    <div class="container">
-    <div class="row">
-
-      
-     <div id="sidebar" class="col-sm-3">
-        
-
-<!-- Top navbar. -->
-    <nav class="navbar navbar-default">
+<head>
```
Comment (klion26) [inline content/zh/contributing/contribute-code.html]: I think you need to revert all these changes to `contribute-code.html`

Comment (highfei2011) [reply content/zh/contributing/contribute-code.html]: Ok,I have restored it to the official source file.

**Comment by highfei2011** [comment]
Replace the old PR with a new one.

**Pull request closed without merging**